stories/wizard/wizard.stories.js

```diff
@@ -0,0 +1,897 @@
+import '../../dist/button.css';
+import '../../dist/icon.css';
+import '../../dist/wizard.css';
+
+export default {
+    title: 'Components/Wizard',
+    parameters: {
+        description: `The wizard guides a user through a long or unfamiliar task by dividing it into sections, revealing information in an easy and digestible way. 
+It consists of a walkthrough screen, where the user is prompted to input required information and upon completing a section, the next sections subsequently follow in a prescribed order; and the summary page, where the form is displayed in read-only mode for assessment and final submission.
+
+
+## Usage
+
+**Use the wizard if:**
+
+- the user has to accomplish a long or unfamiliar task
+- the flow consist of a minimum of 3 and a maximum of 8 steps
+
+**Do not use the wizard if:**
+
+- a task has only 2 steps
+- a task has more than 8 steps
+- the format of the task is familiar to the user
+
+## Types
+
+There are two types of wizard that offer different functionality: **standard** and **branching**.
+
+**Use the standard type if:**
+
+- The total number of steps is known in advance.
+- The number of steps does not change during usage.
+- There is linear progression from one step to the next.
+
+**Use the branching type if:**
+
+- The total number of steps is not known.
+- The number of steps may change during usage.
+- There is non-linear progression. In other words, the user’s choice during one step determines which step comes next.
+- In both types of wizard you can let users skip steps. Label these steps as “Optional”.
+
+
+## Layout
+
+The wizard can be used both in a full-screen layout and in the flexible column layout. There is no navigation from the wizard to a next page. After completion or cancellation, the user is always navigated back to the page the wizard was triggered from.
+
+## Responsive paddings
+
+These modifier classes will add horizontal paddings to the content and can be applied on the <code>fd-wizard\\_\\_progress-bar</code> level and/or on the <code>fd-wizard\\_\\_content</code> level.
+
+
+|  **rem** |  &nbsp;&nbsp;&nbsp;&nbsp;**min-width** |  &nbsp;&nbsp;&nbsp;&nbsp;**max width** |  &nbsp;&nbsp;&nbsp;&nbsp;**modifier class** |
```

Comment:
I missed this, let's capitalize: Min-width, Max-width, Modifier class
To stay aligned with the other tables.